sympy/parsing/autolev/__init__.py

```diff
@@ -1,56 +1,60 @@
 from sympy.external import import_module
 
 
-def parse_autolev(inp, output=None):
-    """Parses Autolev code to SymPy code.
+def parse_autolev(autolev_code, include_numeric=False):
+    """Parses Autolev code (version 4.1) to SymPy code.
 
     Parameters
-    ----------
-    inp: str
-         1. Can be the name of a file containing Autolev code.
-         2. Can be a string containing Autolev code.
+    =========
+    autolev_code : str, any object with a readlines() method (such as a file handle or StringIO)
+        Autolev code...
+    include_numeric : boolean, optional
+        If True NumPy, PyDy, or other numeric code is included for numeric evaluation lines in the Autolev code.
+
+    Returns
+    =======
+    sympy_code : str
+        Equivalent sympy and/or numpy/pydy code as the input code.
 
-    output: str
-            1. Can be the name of an output file to which the SymPy code should be written to.
-            2. Can be the string "print". In this case the SymPy code is written to stdout.
-            3. Can be the string "list". In this it returns a list containing the SymPy code.
-                Each element in the list corresponds to one line of code.
 
     Example (Double Pendulum)
-    -------
+    =========================
+    Suppose this is the file we need to parse:
+    ------------FILE--------------
+    % double_pendulum.al
+    MOTIONVARIABLES' Q{2}', U{2}'
+    CONSTANTS L,M,G
+    NEWTONIAN N
+    FRAMES A,B
+    SIMPROT(N, A, 3, Q1)
+    SIMPROT(N, B, 3, Q2)
+    W_A_N>=U1*N3>
+    W_B_N>=U2*N3>
+    POINT O
+    PARTICLES P,R
+    P_O_P> = L*A1>
+    P_P_R> = L*B1>
+    V_O_N> = 0>
+    V2PTS(N, A, O, P)
+    V2PTS(N, B, P, R)
+    MASS P=M, R=M
+    Q1' = U1
+    Q2' = U2
+    GRAVITY(G*N1>)
+    ZERO = FR() + FRSTAR()
+    KANE()
+    INPUT M=1,G=9.81,L=1
+    INPUT Q1=.1,Q2=.2,U1=0,U2=0
+    INPUT TFINAL=10, INTEGSTP=.01
+    CODE DYNAMICS() some_filename.c
+    ------------FILE--------------
```

Comment:
Does this render correctly when parsed? Don't you need a `::` or `.. code: XXX`?